app/services/marketing_site.rb

```diff
@@ -86,6 +86,14 @@ def self.help_document_capture_tips_url
     ).to_s
   end
 
+  def self.help_document_capture_supported_documents_url
```

Comment:
I didn't notice an equivalent method was already available above:

https://github.com/18F/identity-idp/blob/b62ad695587ad3074b6a6049df10190a8241c673/app/services/marketing_site.rb#L45-L51

I'll take care of this in #5662.